components/insight/SRC/org/openmicroscopy/shoola/env/data/AdminServiceImpl.java

```diff
@@ -872,4 +877,14 @@ public boolean isSystemUser(long userID, String key)
         }
         throw new IllegalArgumentException("Key not valid.");
     }
+
+    /**
+     * Implemented as specified by {@link AdminService}.
+     * @see AdminService#lookupLdapAuthExperimenter(SecurityContext, long)
+     */
+    public String lookupLdapAuthExperimenter(SecurityContext ctx, long userID)
+            throws DSOutOfServiceException, DSAccessException
+    {
+        return gateway.lookupLdapAuthExperimenter(ctx, userID);
```

Comment:
No we have to use it. The boolean flag on user only tells us if LDAP or not but in order to display the details we need the call.
Other option will be to replace the boolean flag by the details i.e. string in DB.